scripts/zones/Carpenters_Landing/mobs/Cryptonberry_Assassin.lua

```diff
@@ -4,36 +4,65 @@
 -- !pos 120.615 -5.457 -390.133 2
 -----------------------------------
 local ID = zones[xi.zone.CARPENTERS_LANDING]
-mixins = { require('scripts/mixins/job_special') }
 -----------------------------------
 ---@type TMobEntity
 local entity = {}
 
+local cryptonberrySpecials =
+{
+    [xi.job.SMN] = xi.jsa.ASTRAL_FLOW,
+    [xi.job.BLM] = xi.jsa.MANAFONT,
+    [xi.job.THF] = xi.jsa.PERFECT_DODGE,
+}
+
 entity.onMobInitialize = function(mob)
     mob:setMobMod(xi.mobMod.IDLE_DESPAWN, 180) -- 3 minutes
 end
 
 entity.onMobSpawn = function(mob)
-    xi.mix.jobSpecial.config(mob, {
-        specials =
-        {
-            {
-                id = xi.jsa.MIJIN_GAKURE,
-                begCode = function(mobArg)
-                    mobArg:messageText(mobArg, ID.text.CRYPTONBERRY_ASSASSIN_2HR)
-                end,
-            },
-        },
-    })
-end
-
-entity.onMobEngage = function(mob, target)
+    mob:addImmunity(xi.immunity.LIGHT_SLEEP)
+    mob:setMod(xi.mod.SLEEPRES, 90)
```

Comment:
Similar to what I posted in your previous PR. SLEEP_MEVA should be used here.

Reply:
> Similar to what I posted in your previous PR. SLEEP_MEVA should be used here.

Thanks for info, I have updated the PR!